frontend/src/concepts/pipelines/content/pipelinesDetails/PipelineDetailsTitle.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Label, Split, SplitItem } from '@patternfly/react-core';
+import { PipelineRunKF } from '~/concepts/pipelines/kfTypes';
+import { computeRunStatus } from '~/concepts/pipelines/content/utils';
+import PipelineRunTypeLabel from '~/concepts/pipelines/content/PipelineRunTypeLabel';
+
+type RunJobTitleProps = {
+  run: PipelineRunKF;
+  statusIcon?: boolean;
+  hasJobReference?: boolean;
+  PipelineRunLabel?: boolean;
```

Comment:
Params are `camelCase` unless it's a component function prop -- of which that would not be `boolean`. Boolean values are also often a pseudo-question... 

Suggest:
```suggestion
  includeTypeLabel?: boolean;
```